docs/content/docs/deployment/adaptive_batch.md

```diff
@@ -0,0 +1,176 @@
+| RightOuter    | ✅              | ❌               |
+
+### Usage
+
+The Adaptive Batch Scheduler defaults to **simultaneously enabling** both compile-time static Broadcast Join and runtime dynamic adaptive Broadcast Join. You can control the timing of the Broadcast Join by configuring [`table.optimizer.adaptive-broadcast-join.strategy`]({{< ref "docs/dev/table/config" >}}#table-optimizer-adaptive-broadcast-join-strategy). For example, you can set the value to be RUNTIME_ONLY so that the adaptive Broadcast Join is effective only at runtime. 
+Additionally, you can adjust the following configuration based on the job's specific requirements:
+- [`table.optimizer.join.broadcast-threshold`]({{< ref "docs/dev/table/config" >}}#table-optimizer-join-broadcast-threshold)：The threshold for the amount of data that can be broadcast. When the memory of the TaskManager is large, this value can be increased appropriately; conversely, it can be decreased if the memory is limited.
+
+### Limitations
+
+- Adaptive Broadcast Join does not support optimization of Join operators contained within MultiInput operators.
+- Adaptive Broadcast Join does not support being enabled simultaneously with [Batch Job Recovery Progress]({{< ref "docs/ops/batch/recovery_from_job_master_failure" >}}). Therefore, after [Batch Job Recovery Progress]({{< ref "docs/ops/batch/recovery_from_job_master_failure" >}}) enabled, Adaptive Broadcast Join will not take effect.
+
+## Adaptive Skewed Join Optimization
+
+In Join queries, when certain keys appear frequently, it may lead to significant variations in the amount of data processed by each Join task. 
+This can lead to a significant reduction in the performance of individual processing tasks, thereby degrading the overall performance of the job. 
+However, because the two input sides of the Join operator are related, the same keyGroup needs to be processed by the same downstream sub-task. 
+Therefore, simply relying on automatic load balancing to solve data skew issues in Join operations is not enough. 
+And Adaptive Skewed Join optimization allows the Join operator to **dynamically split skewed and splittable data partitions** based on runtime input statistics, thereby alleviating the tail latency problem caused by skewed data. 
+
+**To ensure the correctness of the Join semantics**, the Adaptive Skewed Join optimization decides whether the input edges can be dynamically split based on the Join type. 
+The scenarios where splitting is possible are as follows:
+
+| **Join Type** | **Left Input** | **Right Input** |
+|:--------------|:---------------|:----------------|
+| Inner         | ✅              | ✅               |
+| FullOuter     | ✅              | ❌               |
+| Semi          | ✅              | ❌               |
+| Anti          | ✅              | ❌               |
+| LeftOuter     | ❌              | ✅               |
+| RightOuter    | ❌              | ❌               |
```

Comment:
It's better to place LeftOuter/RightOuter and FullOuter together.

Maybe order the types as below:
Inner
LeftOuter
RightOuter
FullOuter
Semi
Anti